dpAutoRigSystem/Modules/dpEye.py

```diff
@@ -624,6 +637,44 @@ def rigModule(self, *args):
                     cmds.connectAttr(self.eyeSpecCtrl+"."+self.langDic[self.langName]['c032_follow'], followSPC+"."+self.fkEyeSubCtrl+"W0", force=True)
                     cmds.connectAttr(self.eyeSpecCtrl+"."+self.langDic[self.langName]['c032_follow'], eyeSpecFollowRev+".inputX", force=True)
                     cmds.connectAttr(eyeSpecFollowRev+".outputX", followSPC+"."+self.baseEyeCtrl+"W1", force=True)
+                    # specular scale control:
+                    self.eyeSpecScaleCtrl = self.ctrls.cvControl("id_091_EyeSpecScale", ctrlName=side+self.userGuideName+"_SpecScale_Ctrl", r=0.2*self.ctrlRadius, d=self.curveDegree)
+                    cmds.delete(cmds.parentConstraint(self.cvSpecularLoc, self.eyeSpecScaleCtrl, maintainOffset=False))
+                    eyeSpecScaleZeroGrp = dpUtils.zeroOut([self.eyeSpecScaleCtrl])
+                    cmds.parent(eyeSpecScaleZeroGrp, self.eyeSpecCtrl)
+                    cmds.parentConstraint(self.eyeSpecScaleCtrl, self.eyeSpecScaleJnt, maintainOffset=False, name=self.eyeSpecScaleJnt+"_PaC")
+                    cmds.scaleConstraint(self.eyeSpecScaleCtrl, self.eyeSpecScaleJnt, maintainOffset=True, name=self.eyeSpecScaleJnt+"_ScC")
+                    # fixing flip mirror:
+                    if s == 1:
+                        if cmds.getAttr(self.moduleGrp+".flip") == 1 or 0:
```

Comment:
Just delete this line 649 and remove the indentation below, because this conditional will be always true.

Reply:
Thank you for the feedback, Danilo!
I've edited the line 649 and it's working. :)